terraform/environments/electronic-monitoring-data/s3.tf

```diff
@@ -31,6 +31,36 @@ resource "aws_s3_bucket" "capita_landing_bucket" {
   bucket = "capita-${random_string.capita_random_string.result}"
 }
 
+resource "aws_s3_bucket_policy" "capita_landing_bucket_policy" {
+  bucket = aws_s3_bucket.capita_landing_bucket.id
+  policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Deny",
+            "Principal": {
+                "AWS": "*"
+            },
+            "Action": "s3:*",
+            "Resource": [
+                "arn:aws:s3:::capita-dyx3807oi4/*",
+                "arn:aws:s3:::capita-dyx3807oi4"
+            ],
+            "Condition": {
+                "Bool": {
+                    "aws:SecureTransport": "true"
```

Comment:
Does this condition permit plain HTTP?
See : second example policy here

https://repost.aws/knowledge-center/s3-enforce-modern-tls

Reply:
I think it does, this statement will match

"Secure Transport is true and TLS is less than 1.2"

.. which means insecure transport will not be matched by this policy and not denied.